test/e2e/scripts/gen_crl_testing_certs.sh

```diff
@@ -0,0 +1,139 @@
+#!/bin/bash -ex
```

Comment:
The script will not run in the pipeline, but we need to keep it for future maintenance as generating those test certificates with CRL is not easy.

Reply:
Can you mention that in the script as a comment?

Reply:
Updated.